api/namex/services/name_request/abstract_name_request.py

```diff
@@ -151,13 +151,25 @@ def generate_nr(cls):
         return nr_num
 
     @classmethod
-    def create_expiry_date(cls, start: datetime, expires_in_days: int, expiry_hour: int = 0, expiry_min: int = 1,
+    def create_expiry_date(cls, start: datetime, expires_in_days: int, expiry_hour: int = 23, expiry_min: int = 59,
```

Comment:
If there is no calling function that sets expiry_hour/min, I would be prefer these parameters to be removed -- it may prevent a future logic bug.

(Same with timezone: if it is always UTC, and should always be UTC, remove the param.)

Reply:
corrected my comment above ^^^

Reply:
fixed